README_SECURITY.md

```diff
@@ -38,8 +38,27 @@ docker run ... -e extender.authentication.platforms=windows,linux,macos extender
 ```
 
 ### Users
-Users are listed in a text file, one username per line followed by the user password, user roles (platform access) and user status ("enabled" or "disabled"). The build script will copy the files in `extender/server/users` to the Docker image when building. You can specify which user file to use in `application.yml` or by passing the `extender.authentication.users` environment variable when starting Docker.
+Users are listed in Java properties format with one username per line followed by the user password, user roles (platform access) and user status ("enabled" or "disabled"). Example:
 
+```
+bob = super5ecret,ROLE_WINDOWS,ROLE_LINUX,ROLE_MACOS,enabled
+may = top5ecret,ROLE_MACOS,enabled
```

Comment:
The "superSecret" and "topSecret" is a bit unclear I think.
Perhaps "password1" and "password2" is better?